database/migrations/0000_00_00_000000_create_families_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateFamiliesTable extends Migration
```

Comment:
What are all these migrations for, exactly? I don't think we store any of this information in EJ databases at the moment.

Reply:
I've looked into this - these migrations are only used for tests, and Laravel does _not_ run them if sdk-php happens to be installed into a project that's using Laravel migrations itself. So I don't think we have to worry too much about them being here.